<ui version="4.0" >
 <class>GroupChatToaster</class>
 <widget class="QWidget" name="GroupChatToaster" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>168</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="windowFrame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <property name="horizontalSpacing" >
       <number>0</number>
      </property>
      <property name="verticalSpacing" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>0</number>
        </property>
        <property name="verticalSpacing" >
         <number>0</number>
        </property>
        <item row="0" column="2" >
         <widget class="QPushButton" name="closeButton" >
          <property name="minimumSize" >
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="icon" >
           <iconset resource="../images.qrc" >:/images/close-down.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="flat" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>225</width>
            <height>22</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>4</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <property name="horizontalSpacing" >
         <number>0</number>
        </property>
        <property name="verticalSpacing" >
         <number>0</number>
        </property>
        <item rowspan="3" row="0" column="0" >
         <widget class="QLabel" name="pixmaplabel" >
          <property name="minimumSize" >
           <size>
            <width>74</width>
            <height>74</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>74</width>
            <height>74</height>
           </size>
          </property>
          <property name="styleSheet" >
           <string>border-image: url(:/images/avatar_background.png);</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="pixmap" >
           <pixmap resource="../images.qrc" >:/images/konversation128.png</pixmap>
          </property>
          <property name="scaledContents" >
           <bool>true</bool>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>121</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" >
         <widget class="QPushButton" name="messageButton" >
          <property name="font" >
           <font/>
          </property>
          <property name="styleSheet" >
           <string>color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #49881F, stop: 1 #49881F );</string>
          </property>
          <property name="text" >
           <string>New Forum Message</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>121</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <layout class="QGridLayout" >
        <property name="topMargin" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>81</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="messageLabel" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>131</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>384</width>
          <height>61</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>messageButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc" />
 </resources>
 <connections/>
</ui>
